gen2-camera-demo/main.py

```diff
@@ -372,9 +393,21 @@ def test_pipeline():
                     continue
                 #print("Received frame:", name, "tstamp", image.getTimestamp().total_seconds())
                 # Skip some streams for now, to reduce CPU load
-                if name in ['left', 'right']: continue
+                if name in ['left', 'right', 'rectified_left', 'rectified_right']: continue
```

Comment:
I think it is better to allow `rectified_left`, `rectified_right` so that users can take a look at them and see how much they are rectified.

Reply:
I did mean to put this back, was testing performance.
This is done @saching13